### PR TITLE
Add Operation Guide

### PR DESCRIPTION
**Pull request by r-gaia-cs**
This replace #12 and address some of the comments from @moorepants and @aterrel.

### CODE REVIEW

File: organization/operations.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# NumFOCUS GSOC Operations
+
+**This is a draft.**
+
+[NumFOCUS](http://numfocus.org/)
+will apply to Google Summer of Code (GSoC)
```
Comment (kynan) [inline organization/operations.md]: Rather: has been approved as a GSoC organisation?

Comment (kynan) [inline organization/operations.md]: Sorry, ignore that, I had misunderstood the purpose of this.

File: organization/operations.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+**Coordinators** and **mentors** should vote **only** in proposals
+related with the project/organization they are involved.
+
+## Guidelines to Select Accept Students Proposals
+
+**Organization administrator** and **coordinators** will
+discuss and accept/reject students proposal based on:
+
+1.  the number of slots received;
```
Comment (moorepants) [inline organization/operations.md]: Google has you request slots after the students have submitted. So you can rank and choose the best applications before knowing how many slots there are. Then you request slots, get some, and have to further refine who's in or out. So this list skips that technical detail but I think it is fine.

File: organization/operations.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+**If** NumFOCUS received few slots than projects/organizations
+under its umbrella,  **organization administrator** and
+**coordinators** will not accept more than one proposal
+per project/organization. For decide which projects/organizations
+receive one slot, **organization administrator** and **coordinators**
+will considere
+
+1.  which project/organization can only apply under NumFOCUS umbrella;
+2.  if the project/organization has enough mentors for the proposal;
+3.  if the project/organization receive proposals from strong candidates.
```
Comment (moorepants) [inline organization/operations.md]: If the organization already has plenty of slots from another umbrella organization.

**Comment by moorepants** [comment]
Looks good. Thanks for drafting it.

**Comment by r-gaia-cs** [comment]
I addressed @moorepants comments and going to merge this.

**Pull request merged**